src/main/java/frc/trigon/robot/misc/PathFollowingController.java

```diff
@@ -0,0 +1,30 @@
+package frc.trigon.robot.misc;
+
+import com.pathplanner.lib.trajectory.PathPlannerTrajectoryState;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.kinematics.ChassisSpeeds;
+import frc.trigon.robot.RobotContainer;
+
+public class PathFollowingController implements com.pathplanner.lib.controllers.PathFollowingController {
```

Comment:
Maybe call it `SwervePathFollowingController` or anything that makes it different than `PathFollowingController`?